src/dodal/devices/vmxm/vmxm_sample_motors.py

```diff
@@ -0,0 +1,7 @@
+from ophyd import Component as Cpt
+from ophyd import EpicsMotor
+from ophyd.epics_motor import MotorBundle
+
+
+class VmxmSampleMotors(MotorBundle):
```

Comment:
We needed this to work around a bug/feature(?) of the VMXm motion system, we needed to manually resend omega in order to get the expected motion.

Add a comment along these lines?